### DISCUSSION TITLE
Homophobic bias in ERNIE ?

### DISCUSSION THREAD

**Discussion by mikotondria** [PaddlePaddle/PaddleFormers#2108] [General]
Will not output anything with 'gay' in prompt.
No accident.
What a terrifyingly mundane and tragic aspect of the past to try to drag into the future.
I'm so embarrassed for you all.

**Comment by 3cha1n** [comment]
![image](https://user-images.githubusercontent.com/96457745/201487524-229f9ba9-b2c7-4a2b-9e33-25c01533bebb.png)
I was very disappointed to read this 😞
I did get an output for "a gay cat", so maybe it depends on how the language model translates "gay". Did you try with a variety of prompts?
